tools/acrntrace/README.rst

```diff
@@ -12,6 +12,67 @@ A ``scripts`` directory includes scripts to analyze the trace data.
 Usage
 *****
 
+The ``acrntrace`` tool runs on the Service OS (SOS) to capture trace data and
+output to trace file under ``/tmp/acrntrace`` with raw (binary) data format.
+
+Options:
+
+-h			print this message
+-t period		specify polling interval in milliseconds [1-999]
+-c			clear the buffered old data
+
+The ``acrntrace_format.py`` is a offline tool for parsing trace data (as output
+by acrntrace) to human-readable formats based on given format.
+
+Here's an explanation of the tool's parameters:
+
+.. code-block:: none
+
+   acrntrace_format.py [options] [formats] [trace_data]
+
+Options:
+
+-h    print this message
+
+- A *formats* file specifies the rules to reformat the *trace_data* collected by
+  ``acrntrace`` into a human-readable text form.  The rules in this file are of
+  the form::
+
+    event_id  text_format_string
+
+  The text_format_string may include format specifiers, such as
+  ``%(cpu)d``, ``%(tsc)d``, ``%(event)d``, ``%(1)d``, and ``%(2)d``.
+  The 'd' format specifier outputs in decimal, alternatively 'x' will
+  output in hexadecimal and 'o' will output in octal.
+
+  These respectively correspond to the CPU number (cpu), timestamp
+  counter (tsc), event ID (event) and the data logged in the trace file.
+  There can be only one such rule for each type of event.
+
+  A pre-defined formats file is available in tools/acrntrace/scripts/formats
```

Comment:
Better would be:

    An example *formats_file* is available in the acrn_hypervisor repo in
    ``hypervisor/tools/acrntrace/scripts/formats``.

Reply:
@dbkinder Will do. Thanks.